Calendar/BookingHelpers/GraphInstaller.ps1

```diff
@@ -0,0 +1,86 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# ===================================================================================================
+# Graph Modules installer Functions
+# ===================================================================================================
+function CheckGraphBookingsModuleInstalled {
+    [CmdletBinding(SupportsShouldProcess=$true)]
+    param ()
+
+    if (Get-Command -Module Microsoft.Graph.Bookings) {
+        Write-Host "Microsoft.Graph.Bookings module is already installed."
+    } else {
+        # This is slow, to the tune of ~10 seconds, but much more complete.
+        # Check if Microsoft.Graph.Bookings module is installed
+        $ModuleInstalled = Get-Module -ListAvailable | Where-Object { $_.Name -eq 'Microsoft.Graph.Bookings' }
+
+        if ($ModuleInstalled) {
+            Write-Host "Microsoft.Graph.Bookings module is already installed."
+        } else {
+            # Check if running with administrator rights
+            $IsAdministrator = ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)
+
+            if (-not $IsAdministrator) {
+                Write-Host "Please run the script as an administrator to install the Microsoft.Graph.Bookings module."
+                exit
```

Comment:
We have a this in our shared code. 

https://github.com/microsoft/CSS-Exchange/blob/main/Shared/Confirm-Administrator.ps1

Reply:
Changed all references in code, added Script Variable IsAdministrator (Referenced in GetBookingdiagnosticsSummary, GraphInstaller and ExcelModuleInstaller)

Reply:
making comments unresolved till the commit is pushed.